From 011e88169c5674eae1ed826754988afcf5a44f77 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 20 Jul 2020 16:23:43 +0800
Subject: [PATCH 1/1] Test case for driver .asm file

Description: Change the '_SystemMemoryEndInit' in the ArmPlatformPkg/PrePi/Arm/ModuleEntryPoint.asm

Repo: edk2

Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 ArmPlatformPkg/PrePi/Arm/ModuleEntryPoint.asm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ArmPlatformPkg/PrePi/Arm/ModuleEntryPoint.asm b/ArmPlatformPkg/PrePi/Arm/ModuleEntryPoint.asm
index 3da789205124..79dd10d85bab 100644
--- a/ArmPlatformPkg/PrePi/Arm/ModuleEntryPoint.asm
+++ b/ArmPlatformPkg/PrePi/Arm/ModuleEntryPoint.asm
@@ -44,7 +44,7 @@ _SetSVCMode
 _SystemMemoryEndInit
   adrll r1, mSystemMemoryEnd
   ldrd  r2, r3, [r1]
-  teq   r3, #0
+  teq   r3, #1
   moveq r1, r2
   mvnne r1, #0
 
-- 
2.27.0.windows.1

